### ISSUE TITLE
aggregate scenarios files of the same feature into the same feature

### ISSUE THREAD

**Issue by xywang68** [gkushang/cucumber-html-reporter#87]
Hello, I hope this qualifies for a feature request.
Suppose I am running scenarios of the same feature file in parallel, and at the end I have created multiple JSON reports of different scenarios from the same feature file.
Now when I generate a cucumber report, I got a report with separate features of the same name, and inside  each feature the scenario of one of the run.
It would be nice to have the scenarios listed under the same feature if the feature is the same (it is from the same feature file).

**Comment by gkushang** [comment]
@xywang68 

It makes sense. Are you currently passing as `jsonDir`?

**Comment by xywang68** [comment]
Not exactly. I am passing as feature with multiple line numbers, such as:
parallel run 1: feature-one:10:20:30
parallel run 2: feature-one:40:60:70
etc.
I will end up with 2 json files:
feature-one_10_20_30.json
feature-one_40_50_60.json
both these json files are processed by the cucumber-html-reporter excepted the same feature name feature-one is listed twice, one with scenario 10, 20 and 30, and once with scenario 40, 50 and 60.
.

**Comment by arjunjh10** [comment]
I am getting the same issue. Running cucumber in Parallel for two different configurations. 
Using this reporter results in the following output: Example
![screen shot 2017-08-07 at 1 42 44 pm](https://user-images.githubusercontent.com/7624561/29009314-57482db2-7b76-11e7-8533-b2d6d077f1f9.png)
its the same feature/scenario run on two different browser configurations. is there a way we could properly segregate them in the html so that we know which config was used for which scenario or a feature providing a better readability to our report?

Also, I was passing the meta-data as well but the run would have only 1 set of the meta-data instead of 2 in my case of 2 parallel runs

**Comment by xywang68** [comment]
Sorry @gkushang , I misunderstood your question. Yes I am passing in as jsonDir.

Here is my command line:
node ${ProjectFullPath}/scripts/generate-reports.js ${ReportDir}
